### PR TITLE
[16.0][OU-ADD] fetchmail: nothing to do

### PR DESCRIPTION
**Pull request by huguesdk**
`fetchmail` was merged into `mail` without model or data changes, and it is already present in the list of merged modules in `apriori.py`.

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -138,7 +138,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | event_sms                                       |                      |No DB layout changes.                            |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| fetchmail                                 |                      |                                                 |
+| |del| fetchmail                                 | Nothing to do        | Merged into mail.                               |
```
Comment (legalsylvain) [inline docsource/modules150-160.rst]:
```suggestion
| |del| fetchmail                                 | Done        | Merged into mail.                               |
```

(as something is done in the apriori.py file).

**Review by legalsylvain — COMMENTED** [review]
minor remarks. Otherwise, LGTM.

File: docsource/modules150-160.rst (hunk 1 above)
Comment (huguesdk) [reply docsource/modules150-160.rst]: thanks for the review @legalsylvain! fixed and updated git comment.

**Review by legalsylvain — APPROVED** [review]
/ocabot merge nobump

**Comment by legalsylvain** [comment]
@huguesdk Could you rebase ?

**Comment by huguesdk** [comment]
sure, but why is it needed, as there are no conflicts? the ci failed during `pre-commit` with the error “premature close”, as if the process was somehow aborted.

**Comment by legalsylvain** [comment]
let's retry.

/ocabot merge nobump

**Pull request merged**